### ISSUE TITLE
Girder Command Line section for Folders/Items(/Files?)

### ISSUE THREAD

**Issue by kotfic** [girder/girder#2693]
So I was staring at the spiffy new Gitlab container registry and really  appreciating that it has a lot of boxes that are like "run this command  to pull this container" or  "run this command to login to the registry"  and it struck me that it would be totally SUPER if girder had this same kind of functionality.   Obviously it wouldn't be appropriate for all  instances so you'd want it to be a plugin,  but i can imagine (especially around upload/download)  this being really useful if you, like me,  often find yourself needing to upload something from somewhere other than your local machine.                                         
                                                                                                       
 I was thinking about a section under the metadata area called "Command line" on Folders/Items that would say something like:                                                 
                                                                                                       
### Download this Folder with the cli                                                                   
 ```shell
girder-client 
   --token=N7LxAWIotBDFc1h3hNOvK2FptIdQk9XbnXWW7Qdw5ZWWKIdcPtHO0eObysJzwM5l \
   --api-url=https://data.kitware.com/api/v1 \  
   download 56054acc8d777f6ddc3da9fe [/path/to/folder] 
```                                                                                           
                                                                                                       
 ### Upload a Folder here with the cli
```shell                                                                   
 girder-client  
  --token=N7LxAWIotBDFc1h3hNOvK2FptIdQk9XbnXWW7Qdw5ZWWKIdcPtHO0eObysJzwM5l \  
  --api-url=https://data.kitware.com/api/v1 \ 
 upload 56054acc8d777f6ddc3da9fe [/path/to/folder] 
```                                                                                            
 Maybe with little a little "copy to clipboard" icon so I could just copy the command (sans the [/path/to/folder]) and run it?

**Comment by brianhelba** [comment]
I don't think this needs to be a plugin if the UI is appropriately non-intrusive and non-confusing to less-CLI-sophisticated users.

**Comment by mgrauer** [comment]
Which token should it choose?

**Comment by mgrauer** [comment]
Somewhat related to #1637. At least we could do them consistently or have general support that helps both.

**Comment by kotfic** [comment]
I would say return the token from ```token/current```  That is the same token the browser is using to maintain the session,  and then use the [newly minted](https://github.com/girder/girder/pull/2689) ```--token``` option to girder-client
